recipes/pbzip2/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "1.1.13" %}
+{% set major_version = "1.1" %}
+{% set sha1 = "f61e65a7616a3492815d18689c202d0685fe167d" %}
+{% set md5 = "4cb87da2dba05540afce162f34b3a9a6" %}
+
+package:
+  name: pbzip2
+  version: {{ version }}
+
+source:
+  fn: pbzip2-{{ version }}.tar.gz
+  url: https://launchpad.net/pbzip2/{{ major_version }}/{{ version }}/+download/pbzip2-{{ version }}.tar.gz
+  sha1: {{ sha1 }}
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: 'make PREFIX=$PREFIX && make install PREFIX=$PREFIX'
```

Comment:
Also, is there a `make check` we can run? Ideally, when the build script is getting complex, we prefer the `build.sh` file for clarity and maintainability.